### ISSUE TITLE
Advanced Panel - Custom CSS: Fix bottom pane so that it stays in the viewport.

### ISSUE THREAD

**Issue by MrMasterKey** [soflyy/oxygen-bugs-and-features#3522]
**Describe the feature you'd like to see included in Oxygen.**
It's halfway between bug and feature. In your Editor panel, Advanced, Custom CSS, the bottom pane moves with the code. It should stay fixed at the bottom, so that changes to the CSS can be applied more easily without first having to scroll that pane into the viewport.

**What are the use cases for this feature?**
Better UX, less time wasted scrolling.

**Examples of this feature or functionality.**
N/A

![Screenshot 2024-03-23 at 10 45 26](https://github.com/soflyy/oxygen-bugs-and-features/assets/58811173/208a4ce2-1294-424c-9197-63c30f1a1ac6)
![Screenshot 2024-03-23 at 10 46 25](https://github.com/soflyy/oxygen-bugs-and-features/assets/58811173/463c1945-6143-4eb4-8b31-0a9a0c71f01c)

**Comment by Warren-WordPress** [comment]
yep, I know what you mean.  Using the browser keystrokes to zoom out and find the missing GUI buttons has become a part of life.  Definitely a quality of life improvement that would get my vote.
